flycheck.el

```diff
@@ -5351,6 +5353,30 @@ See URL `http://phpmd.org/' for more information about phpmd."
                       errors)))))))))
        (nreverse errors)))))
 
+(defun flycheck-parse-tslint (output checker buffer)
+  "Parse TSLint errors from JSON OUTPUT.
+
+CHECKER and BUFFER denoted the CHECKER that returned OUTPUT and
+the BUFFER that was checked respectively.
+
+See URL `https://palantir.github.io/tslint/' for more information
+about TSLint."
+  (let ((json-object-type 'plist))
+    (let ((errors)
+	  (tslint-json-output (json-read-from-string output)))
```

Comment:
The indentation looks a little strange to me.  Are there tabs in this file?  Can you try to reindent the entire function?